FAQ.md

```diff
@@ -327,20 +365,72 @@ You can scale TiDB as your business grows.
 
 ### Monitor
 
-Should I deploy the TiDB monitoring framework (Prometheus + Grafana) on a standalone machine or on multiple machines? What is the recommended CPU and memory?
+#### Should I deploy the TiDB monitoring framework (Prometheus + Grafana) on a standalone machine or on multiple machines? What is the recommended CPU and memory?
 
 The monitoring machine is recommended to use standalone deployment. It is recommended to use 8 core CPU, 32 GB+ memory and 500 GB+ hard disk.  
 
 #### The monitor can't show all the metrics.
 
 Check the time difference between the machine time of the monitor and the time within the cluster. If it is large, you can correct the time and the monitor will display all the metrics.
 
+#### How to configure to monitor Syncer status?
+
+Download [Syncer Json](https://github.com/pingcap/docs/blob/master/etc/Syncer.json) to Grafana, modify the Prometheus configuration file, add the following:
+
+```
+- job_name: ‘syncer_ops’ // task name
+    static_configs:
+      - targets: [’10.10.1.1:10096’] // Syncer monitoring address and port, informing Prometheus to pull the data of Syncer
+```
+
+Restart Prometheus. 
+
 ### Migrate
 
 #### Can a MySQL application be migrated to TiDB?
 
 Yes. Your applications can be migrated to TiDB without changing a single line of code in most cases. You can use [checker](https://github.com/pingcap/tidb-tools/tree/master/checker) to check whether the Schema in MySQL is compatible with TiDB.
 
+#### Accidentally import the MySQL user table into TiDB and cannot login, how to restore?
+
+Restart the TiDB service, add the `-skip-grant-table=true` parameter in the configuration file. Login the cluster and rebuild using the following SQL statements:
```

Comment:
rebuild using the following SQL statements: -> recreate the `mysql.user` table using the following statement: